app/models/roster.rb

```diff
@@ -4,10 +4,12 @@
 
 class Roster < ApplicationRecord
   include PhoneHelper
+  extend FriendlyId
 
   has_paper_trail
-  has_many :assignments, dependent: :destroy
+  friendly_id :name, use: %i[slugged finders]
```

Comment:
Are you using `:finders` just so you don't have to use the `friendly` scope? It seems to me if we're setting it up for the first time, we should use their current defaults rather than an old version's defaults.

Reply:
Yeah, the duck-type-aficionado in me kinda likes that both things count as roster "id"s and you don't have to change as much code. And, when I [first read the comments in the initializer](https://github.com/norman/friendly_id/blob/a72fac90282ad1e95611eb4bb381dd9d6702209b/lib/friendly_id/initializer.rb#L39-L45) (which I pruned down a bit), it sounded like it was along the lines of "it might be incompatible with your data, but it's more performant."

On second read, it sounds like the gem authors maybe wish they hadn't done it that way? Or at least there's disagreement among said developers. I think I've come around to adding explicit `.friendly`s, though.